packages/jest-bolt-receiver/README.md

```diff
@@ -0,0 +1,80 @@
+# Jest Bolt Receiver
+
+A [receiver](https://slack.dev/bolt/concepts#receiver) built for testing Slack Bolt apps with Jest. This receiver allows you to easily send mock events to your app and write consistent tests.
+
+## Table of Contents
+
+- [Install](#install)
+- [Usage](#usage)
+  - [Sending Events](#sending-events)
+
+## Install
+
+```bash
+yarn add --dev @slack-wrench/jest-bolt-receiver
+# or
+npm install --save-dev @slack-wrench/jest-bolt-receiver
+```
+
+## Usage
+
+```typescript
+import JestReceiver from '@slack-wrench/jest-bolt-receiver';
+import delay from 'delay';
+
+// Import your app that creates a Slack Bolt App.
+import MyApp from './app';
+
+describe('My Awesome App', () => {
+  let receiver: JestReceiver;
+  let app: MyApp;
+
+  beforeEach(() => {
+    receiver = new JestReceiver();
+    // Pass in this receiver instead of yours or the default express one
+    app = new MyApp({ receiver });
+  });
+
+  it('Can handle a slash command', async () => {
+    receiver.sendSlashCommand('/my-apps-command', { text: 'I called it!' });
+    await delay(0); // Wait for anything async.
+
+    // Test what should have happened.
+    expect();
+  });
```

Comment:
might it be better to show a real basic test here? This doesn't demonstrate the value here.